homeassistant/components/securitas_direct/__init__.py

```diff
@@ -0,0 +1,114 @@
+"""Support for Securitas Direct (AKA Verisure EU) alarm control panels."""
+
+from datetime import timedelta
+
+import homeassistant.helpers.config_validation as cv
+import voluptuous as vol
+from homeassistant.const import (
+    CONF_CODE,
+    CONF_PASSWORD,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+    EVENT_HOMEASSISTANT_STOP,
+)
+from homeassistant.helpers import discovery
+from homeassistant.util import Throttle
+from pysecuritas.api.alarm import Alarm
+from pysecuritas.api.installation import Installation
+from pysecuritas.core.session import Session
+
+CONF_COUNTRY = "country"
+CONF_LANG = "lang"
+CONF_INSTALLATION = "installation"
+DOMAIN = "securitas_direct"
+MIN_SCAN_INTERVAL = timedelta(seconds=20)
+DEFAULT_SCAN_INTERVAL = timedelta(seconds=40)
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
Please use our scaffold script to generate best practice code:
https://developers.home-assistant.io/docs/creating_component_index